contributors/guide/pull-requests.md

```diff
@@ -249,6 +250,31 @@ Likewise, if Feature-X is similar in form to Feature-W which was checked in last
 Feature-X.
 (Do that in its own commit or pull request, please.)
 
+## Don't Open Pull Requests That Span the Whole Tree
+
+Often a new contributor will find some problem that exists in many places across the main
+Kubernetes tree, and file a PR to fix it everywhere at once. Maybe there's a cool new
+function in the latest golang release that everyone ought to be using, or a
+recently-deprecated function that ought to be replaced with calls to its replacement.
+Sometimes a contributor will run a linter or security scanner across the code to find
+problems, or fix a particular spelling mistake in comments or variable names. (It's
+"deprecated", not "depreciated"!)
+
+The problem with this approach is that different parts of the Kubernetes tree are owned by
+different SIGs, and so changes to those different parts require approvals from different
+people. A PR containing 20 one-line changes scattered across the tree could end up needing
+5 or 10 approvals or more before it can be merged. (While there are top-level approvers
```

Comment:
and the project is trying to move away from top-level approvers https://github.com/kubernetes/kubernetes/issues/112867#issuecomment-1268722455

Reply:
Well, in fact, `pkg/OWNERS` and `cmd/OWNERS` both have `no_parent_owners: true` (as fallout from that issue). But they also have roughly the same set of people that used to be in the top-level `OWNERS` so the overall effect (for PRs that only change code) is still the same...

Reply:
yeah, I just want to highlight that in the future this PR will be still more important, since the trend is to avoid that